### PR TITLE
initial dashboard

### PR DESCRIPTION
**Pull request by emmayuan25**

### CODE REVIEW

File: src/main/webapp/javascript/dashboard.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+let username = sessionStorage.getItem("username");
+
+if(username !== ""){
+    fetchTrips(username);
+}
+
+function fetchTrips(username) {
+    fetch('/get-trips').then(response => response.json()).then((trips) => {
```
Comment (sbahaud) [inline src/main/webapp/javascript/dashboard.js:8]: Don't you have to pass in the userId here?

File: src/main/webapp/javascript/dashboard.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+let username = sessionStorage.getItem("username");
+
```
Comment (sbahaud) [inline src/main/webapp/javascript/dashboard.js:2]: It might be better to store the userId instead of username, since that is what the backend wants

**Review by sbahaud — APPROVED** [review]
Left some comments

File: src/main/webapp/javascript/dashboard.js (hunk 2 above)
Comment (emmayuan25) [reply src/main/webapp/javascript/dashboard.js:2]: The log in is still having trouble, we'll store userid after resolving that

**Pull request merged**